conda.bzl

```diff
@@ -20,12 +20,32 @@ CONDA_SHA = {
 }
```

Comment:
Also please run [`buildifier`](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md) at the end if you haven't already

Reply:
Done.